### PR TITLE
feat: added umbrella-off icon

### PR DESCRIPTION
**Pull request by jguddas**
## What is the purpose of this pull request?

- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Icon use case 

Unsecure / unprotected / without antivirus.

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [ ] The icons are solely my own creation.
- [ ] The icons were originally created in #<issueNumber> by @<githubUser>
- [x] I've based them on the following Lucide icons: umbrella
- [ ] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to two points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Review by ericfennis — COMMENTED** [review]
Hmm find the usecase a bit weak. For antivirus I will think more about the `shield` icon.

**Comment by karsa-mistmere** [comment]
I like this one in general, although I share Eric's scepticism about use cases.

Some further use cases I could come up with, all signifying the concept of "uncovered" either literally and figuratively, e.g.:
* to signify that a venue is outdoors and has no protection from the elements
* in insurance services, that a particular service or item is not covered by the policy
* in finance apps that investment or financial decision is risky and not protected

None of these feel super strong, but I'm okay with merging this if you think these use cases warrant it a spot. :)

File: icons/umbrella-off.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "colebemis",
+    "csandman",
+    "ericfennis",
+    "karsa-mistmere",
+    "jguddas",
+    "jguddas"
+  ],
+  "tags": [
+    "rain",
+    "weather"
```
Comment (jguddas) [inline icons/umbrella-off.json:13]:
```suggestion
    "weather",
    "uncovered",
    "uninsured",
    "antivirus",
    "unprotected",
    "risky"
```

**Pull request merged**